nxcomp/Loop.cpp

```diff
@@ -671,15 +671,11 @@ static int ReopenLogFile(char *name, ostream *&stream, int limit);
 static void handleCheckSessionInLoop();
 static void handleCheckBitrateInLoop();
 
-#if defined(TEST) || defined(INFO)
```

Comment:
Because the logging code is not guarded by these tests anymore, so the functions must always be defined to make the code compile.